### PR TITLE
Gordyd/notes footer link

### PR DESCRIPTION
**Pull request by GordyD**

### CODE REVIEW

**Comment by GordyD** [comment]
@jebeck I have updated this with the new Retina quality image.

**Comment by jebeck** [comment]
This is now out-of-date with our new env (notes.tidepool.io is no longer the right link). Is there an update you would like to see @brandonarbiter?

**Comment by GordyD** [comment]
Any update on this @brandonarbiter?

**Comment by GordyD** [comment]
I have updated this branch with master and ported changes to new underlying file structure -> mainly to do with app.js moving from components.

**Comment by jebeck** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;yemDk9OV&#x2F;21-blip-add-blip-notes-asset-and-its-accompanying-footer-link"> [blip] add  blip notes asset and its accompanying footer link</a>

**Comment by krystophv** [comment]
Now that the Blip Notes link goes directly to a page on the tidepool.org website, do we need the 4 image resources any longer?

**Comment by GordyD** [comment]
Good point. Will remove.

File: app/pages/app/app.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ import async from 'async';
 import sundial from 'sundial';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { Link } from 'react-router';
```
Comment (krystophv) [inline app/pages/app/app.js]: I think this ended up not getting used, might not need to import it here.

**Comment by krystophv** [comment]
Aside from the extraneous `Link` import, this LGTM

**Pull request merged**